### ISSUE TITLE
Fast Lane: active weapon palette

### ISSUE THREAD

**Issue by jotego** [jotego/jtcontra#13]
The weapon selection bar does not indicate which weapon is active: wrong palette?

**Comment by jotego** [comment]
I don't have the PCB, only a picture. But it has so few chips and they can only be connected in so few ways, that I'm rather confident about the core being mostly right. About the palette, I tried today another way of connecting it which makes sense too. I used the palette PROM for the tiles, instead of the sprites.
When I did so, the active weapon is clearly identified now in red. But all the score characters flash in colour. Still, it looks better than the previous version. So I am updated the files in JTBIN.
It is possible that the board connects some lines in a twisted way, and obtains different colours. But I don't have the board and I haven't found any video from the real game (all I can see are MAME videos). So this could still be wrong, but I'm betting it's right now so I'm closing it for now.
